Answers.md

```diff
@@ -0,0 +1,21 @@
+Describe the biggest difference between .forEach & .map.
+
+Answer= forEach calls back each element in an array, map returns a new array that is the same size.
```

Comment:
Expand on this.... forEach can also act upon those elements and map can also apply a function to each element in the new array